src/by/epam/lab/Subject.java

```diff
@@ -0,0 +1,53 @@
+package by.epam.lab;
+
+public class Subject {
+    private String name;
+    private double volume;
+    private Material material;
+
+    public Subject() {
+    }
+
+    public Subject(String name, Double volume, Material material) {
+        this.name = name;
+        this.volume = volume;
+        this.material = material;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Double getVolume() {
+        return volume;
+    }
+
+    public void setVolume(Double volume) {
+        this.volume = volume;
+    }
+
+    public Material getMaterial() {
+        return material;
+    }
+
+    public void setMaterial(Material material) {
+        this.material = material;
+    }
+
+    @Override
+    public String toString() {
+        return name + ";" +
+                material + ";" +
+                volume + ";" +
+                getMass();
+    }
+
+    public Double getMass () {
```

Comment:
```suggestion
    public double getMass () {
```
не исправил мои замечания из прошлого ревью ((